lib/pages/activities/activity_page.dart

```diff
@@ -73,11 +73,7 @@ class ActivityPage extends StatelessWidget {
                 itemBuilder: (context, index) {
                   final activity = _activities[index];
                   return ActivityCard(
-                    taskName: activity.taskName,
-                    courseName: activity.courseName,
-                    remainingTime: activity.remainingTime,
-                    topicTheme: activity.topicTheme,
-                    backgroundColor: activity.backgroundColor,
+                    activity: activity,
```

Comment:
We don't need the `ActivityCard` anymore, just use `InfoCard` directly and delete the other one.

Reply:
Done!